src/tools/miri/src/bin/log/tracing_chrome.rs

```diff
@@ -497,31 +519,39 @@ where
         }
     }
 
-    fn get_root_id(span: SpanRef<S>) -> u64 {
-        span.scope()
-            .from_root()
-            .take(1)
-            .next()
-            .unwrap_or(span)
-            .id()
-            .into_u64()
+    fn get_root_id(&self, span: SpanRef<S>) -> Option<u64> {
+        match self.trace_style {
+            TraceStyle::Threaded => {
+                if span.fields().field("tracing_separate_line").is_some() {
+                    // put spans with argument "tracing_separate_line" on a separate trace line
+                    // by setting their "id", see
+                    // https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview#heading=h.jh64i9l3vwa1
+                    Some(span.id().into_u64())
```

Comment:
Maybe `tracing_root` or so would be a better name?

`tracing_separate_line` was just the first thing that came to my mind without understanding anything about what actually happens or what the tracing terminology is.

Reply:
Let's discuss that on Zulip, this PR is an improvement on its own already.